storybook/index.html

```diff
@@ -26,6 +26,20 @@ <h2 class="sbHeading">Modules</h2>
                 <ul class="sbNavList"><!-- list items will be added dynamically --></ul>
             </nav>
             <main class="l-main">
+                <section class="demo" data-module-name="carousel">
+                    <div class="carousel">
+                        <div class="carousel--items">
+                            <img class="carousel--item" src="https://p.bigstockphoto.com/GeFvQkBbSLaMdpKXF1Zv_bigstock-Aerial-View-Of-Blue-Lakes-And--227291596.jpg">
+                            <img class="carousel--item" src="https://media.istockphoto.com/photos/child-hands-formig-heart-shape-picture-id951945718?k=6&m=951945718&s=612x612&w=0&h=ih-N7RytxrTfhDyvyTQCA5q5xKoJToKSYgdsJ_mHrv0=">
+                            <img class="carousel--item" src="https://cdn.eso.org/images/thumb300y/eso1907a.jpg">
+                            <video class="carousel--item" controls=""><source src="https://scontent-sjc3-1.cdninstagram.com/v/t50.2886-16/121085909_350486359401996_35227211841975188_n.mp4?_nc_ht=scontent-sjc3-1.cdninstagram.com&amp;_nc_cat=100&amp;_nc_ohc=Qhu4HTk8eTAAX85T9k-&amp;oe=5FB6E069&amp;oh=3da3a86206a88a315a915a02554ed77c"></video>
```

Comment:
feel free to use whatever you like here to replace the broken one, but the URL I dug up was: https://pic.pikbest.com/19/44/20/95s888piCfpc.mp4